repos/system_upgrade/el8toel9/actors/checkifcfg/libraries/ifcfg.py

```diff
@@ -0,0 +1,145 @@
+import os
+
+from leapp import reporting
+
+
+def process():
+    TRUE_VALUES = ['yes', 'true', '1']
+    GOOD_TYPES = [
+        'ethernet', 'ctc', 'wireless', 'infiniband', 'bond',
```

Comment:
wireless with WEP also needs to inhibit the upgrade

Reply:
team and wireless require plugins, how is this going to be handled?

Reply:
> wireless with WEP also needs to inhibit the upgrade

This actor deals with ifcfg files. WEP is entirely out of scope here.

> team and wireless require plugins, how is this going to be handled?

They're presumably already installed in the system that's being upgraded. Why else would the configuration be there.